templates/_worker.js/index.ts

```diff
@@ -147,9 +149,19 @@ export default {
       for (const matcher of matchers) {
         if (matcher.regexp) {
           const regexp = new RegExp(matcher?.regexp);
+          const nextPathname = __BASE_PATH__
+            ? // Remove basePath from URL, also squish `//` into `/`
+              // If the baseUrl is set to "/docs" the following will happen:
+              // `/docs/profile/settings` -> `/profile/settings`
+              // `/docs` -> `/`
+              // `/docs/` -> `/`
+              // `/docs/_next/static/main.js` -> `/_next/static/main.js`
+              pathname.replace(__BASE_PATH__, "/").replace("//", "/")
```

Comment:
what do you think? shall we also check that __BASE_PATH__ is at the start of the string? (I think it wouldn't really make a difference but I figured I would just ask 🙂)

Reply:
Happy change the logic to "If the pathname startsWith basePath". I think it'll be a tiny bit safer